lib/matplotlib/tests/test_ticker.py

```diff
@@ -296,59 +310,60 @@ def get_view_interval(self):
          range(6), ['0.1', '1e-05', '3.14', '0.2', '-0.2', '-1e-05']),
         (False, 50, np.array([3, 5, 12, 42], dtype='float'), range(6),
          ['3', '5', '12', '42']),
-    ])
-@pytest.mark.parametrize('base', [2.0, 5.0, 10.0, np.pi, np.e])
-def test_LogFormatterExponent(labelOnlyBase, base, exponent,
-                              locs, positions, expected):
-    formatter = mticker.LogFormatterExponent(base=base,
-                                             labelOnlyBase=labelOnlyBase)
-    formatter.axis = FakeAxis(1, base**exponent)
-    vals = base**locs
-    labels = [formatter(x, pos) for (x, pos) in zip(vals, positions)]
-    assert labels == expected
-
-
-def test_LogFormatterExponent_blank():
-    # Should be a blank string for non-integer powers if labelOnlyBase=True
-    formatter = mticker.LogFormatterExponent(base=10, labelOnlyBase=True)
-    formatter.axis = FakeAxis()
-    assert formatter(10**0.1) == ''
-
-
-@pytest.mark.style('default')
-def test_LogFormatterSciNotation():
-    test_cases = {
-        10: (
-             (-1, '${-10^{0}}$'),
-             (1e-05, '${10^{-5}}$'),
-             (1, '${10^{0}}$'),
-             (100000, '${10^{5}}$'),
-             (2e-05, '${2\\times10^{-5}}$'),
-             (2, '${2\\times10^{0}}$'),
-             (200000, '${2\\times10^{5}}$'),
-             (5e-05, '${5\\times10^{-5}}$'),
-             (5, '${5\\times10^{0}}$'),
-             (500000, '${5\\times10^{5}}$'),
-        ),
-        2: (
-            (0.03125, '${2^{-5}}$'),
-            (1, '${2^{0}}$'),
-            (32, '${2^{5}}$'),
-            (0.0375, '${1.2\\times2^{-5}}$'),
-            (1.2, '${1.2\\times2^{0}}$'),
-            (38.4, '${1.2\\times2^{5}}$'),
-        )
-    }
-
-    for base in test_cases:
+    ]
+
+    base_data = [2.0, 5.0, 10.0, np.pi, np.e]
+
+    @pytest.mark.parametrize(
+            'labelOnlyBase, exponent, locs, positions, expected', param_data)
+    @pytest.mark.parametrize('base', base_data)
+    def test_LogFormatterExponent(self, labelOnlyBase, base, exponent,
+                                  locs, positions, expected):
+        formatter = mticker.LogFormatterExponent(base=base,
+                                                 labelOnlyBase=labelOnlyBase)
+        formatter.axis = FakeAxis(1, base**exponent)
+        vals = base**locs
+        labels = [formatter(x, pos) for (x, pos) in zip(vals, positions)]
+        assert labels == expected
+
+    def test_blank(self):
+        # Should be a blank string for non-integer powers if labelOnlyBase=True
+        formatter = mticker.LogFormatterExponent(base=10, labelOnlyBase=True)
+        formatter.axis = FakeAxis()
+        assert formatter(10**0.1) == ''
+
+
+class TestLogFormatterSciNotation(object):
+    testdata = [
```

Comment:
We'll see how the tests go, then, but I'd guess this shouldn't be a problem since it isn't callable.